parse/TechsParser.cpp

```diff
@@ -116,14 +116,14 @@ namespace {
 
             prerequisites
                 =   parse::label(Prerequisites_token)
-                >  (    '[' > +tok.string [ insert(_r1, _1) ] > ']'
+                >  (    ('[' > +tok.string [ insert(_r1, _1) ] > ']')
```

Comment:
What is supposed to be aligning with what here?  Is the newly bracketed clause supposed to be on its own line?